samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,75 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+import click
+
+from samcli.commands.check.resources.Graph import Graph
+from samcli.commands.check.resources.LambdaFunction import LambdaFunction
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class BottleNecks:
+    def __init__(self, graph: Graph):
+        self._graph = graph
+        self._lambda_max_duration = 900000
+
+    def ask_entry_point_question(self):
```

Comment:
Let's add return types for functions as well even if they are None. Functions without annotations will default to `Any` return type.
```
def ask_entry_point_question(self) -> None:
```
https://www.python.org/dev/peps/pep-0484/#the-meaning-of-annotations
> It is recommended but not required that checked functions have annotations for all arguments and the return type. For a checked function, the default annotation for arguments and for the return type is Any.